### PR TITLE
Feature/remove zerorpc

### PR DESCRIPTION
**Pull request by urugang**
1) remove zerorpc
2) add eslint

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -16,16 +16,8 @@
     "experimentalDecorators": true,
     "allowSyntheticDefaultImports": true,
     "forceConsistentCasingInFileNames": true,
-    "typeRoots": [
-      "node_modules/@0x/typescript-typings/types",
```
Comment (kaichen) [inline tsconfig.json:20]: remove this line would broke the typescript build

File: app/mmConfig.js [hunk 2]
```diff
@@ -1,23 +1,18 @@
+require('dotenv').config()
```
Comment (kaichen) [inline app/mmConfig.js:1]: don't load the dotenv file by default

File: app/mmConfig.js [hunk 3]
```diff
@@ -1,23 +1,18 @@
...
   // Tokenlon server address
   EXCHANGE_URL: process.env.EXCHANGE_URL,
   PROVIDER_URL: process.env.PROVIDER_URL,
 
-  //
   // Wallet
   WALLET_ADDRESS: process.env.WALLET_ADDRESS,
-  USE_KEYSTORE: true,
+  USE_KEYSTORE: !process.env.USE_KEYSTORE || JSON.parse(process.env.USE_KEYSTORE),
   WALLET_KEYSTORE: {},
```
Comment (kaichen) [inline app/mmConfig.js:10]: here is the public version for market maker, this option MUST on by default.

File: app/mmConfig.js (hunk 2 above)
Comment (urugang) [reply app/mmConfig.js:1]: removed

File: app/mmConfig.js (hunk 3 above)
Comment (urugang) [reply app/mmConfig.js:10]: removed

File: tsconfig.json (hunk 1 above)
Comment (urugang) [reply tsconfig.json:20]: 去掉后可以正常编译的

**Comment by kaichen** [comment]
Please move the zeroRPC quoter code to `app` folder, still someone need it.

**Comment by kaichen** [comment]
Please move the zeroRPC quoter code to `app` folder, still someone need it.

**Pull request merged**